lib/ui/shared/text_styles.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:hng/ui/shared/colors.dart';
+
+class ZuriTextStyle {
```

Comment:
Also can we just make this not have a class and have all those properties be constant.
It's better to just use 
```
Text(
style: smallLight,
)
```

Than having to

`ZuriTextStyle.smallLight`

Reply:
This was a collective refactor 
The main reason why we went with this format is for font style flexibility(change the typeface and reflects everywhere) and ease to use 
Changing the styles in the ZuriTextStyle to constant would only allow constant values to be inserted which might not be compatible with our colours. I would need to contact the team lead so as to do project-wide refactoring.